internal/convert/process_config_test.go

```diff
@@ -116,3 +113,166 @@ func TestNewReplicaSetProcessConfig(t *testing.T) {
 
 	assert.Equal(t, expected, result)
 }
+
+func Test_newConfigRSProcess(t *testing.T) {
```

Comment:
I added a test of ops manager -> mcli but not one mcli -> ops manager, so now we have both